prime-router/settings/organizations.yml

```diff
@@ -2565,7 +2573,7 @@
         initialTime: 00:00
       translation:
         type: CUSTOM
-        schemaName: none
+        schemaName: "metadata/hl7_mapping/STLTs/ignore/ignore"
```

Comment:
Set this schema to the ORU_R01 and remove the IGNORE schema

Reply:
done